modules/benchmark/R/start.bm.ensemble.R

```diff
@@ -1,21 +1,44 @@
 ##-------------------------------------------------------------------------------------------------#
-##' Start Benchmark Run
+##' Start Benchmark Ensemble
 ##'  
-##' @name start.bm.run
-##' @title Start benchmark run
-##' @param benchmark_reference_run.id,
-##' @param machine.id
-##' @export 
+##' @name start.bm.ensemble
+##' @title Start benchmark ensemble
+##' @param benchmark_reference_run.id
+##' @param con database connection
+##' @export start.bm.ensemble
 ##' 
 ##' @author Betsy Cowdery 
 
-start.bm.run <- function(BRR.id, database, machine.id){ #- leverage workflow.R
+start.bm.ensemble <- function(BRR, con){ 
+  
+    old.settings <- file.path(BRR$settings, "pecan.xml")
+    new.settings <<- sprintf("%s/%s.pecan.xml",BRR$settings, basename(BRR$settings))
+    clean.settings(old.settings,new.settings)
+    
+    commandArgs <- function(trailingOnly = TRUE) new.settings 
+    source("web/workflow.R")
```

Comment:
fragile. Only works if pwd is the pecan folder.  If you really do need to source workflow.R (which you probably will until @rykelly generalizes it a bit more) you need to get the PEcAn directory from elsewhere and paste that together with this bit.